ocaml/testsuite/tests/mixed-blocks/generated_byte_test.ml

```diff
@@ -356,6 +356,1171 @@ type t296 = { str0 : string; mutable float1 : float; float_u2 : float# }
 type t297 = { mutable float0 : float; mutable str1 : string; mutable str2 : string; mutable str3 : string; mutable float_u4 : float#; mutable float_u5 : float# }
 type t298 = { mutable str0 : string; mutable float1 : float; mutable str2 : string; float_u3 : float#; mutable imm4 : int }
 type t299 = { float0 : float; mutable imm1 : int; mutable float_u2 : float#; mutable imm3 : int }
+type t300 =
```

Comment:
(It actually grew from ~5000 lines to ~13,000.)

I think it is easy to control the size of this file. There are knobs in the `test.ml` file that let you say "I want to test fewer combinations". 

> But probably we'll add more things that add more dimensions for it to grow exponentially on later 

I don't think it will be exponential. I think it will be more like adding 5,000 more lines. But again it is easy to reduce this (and easy for me to run the test to gauge whether it's taking too long.

>  Should we consider splitting it up into multiple files or testing fewer combinations?

I don't think so at this point in time.

Reply:
Fair enough!